packages/ComponentLibrary/src/components/Table/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface Organization {
```

Comment:
This interface must not be included in the main component library folder. If it is to mock a test, create a folder for this use.